examples/icon/data/location-icon-mapping.json

```diff
@@ -0,0 +1,154 @@
+{
```

Comment:
This is a duplicate. Copying static files from external directories when building demo can be messy.